e2e/semverutil/semver.go

```diff
@@ -16,6 +18,17 @@ type FeatureReleases struct {
 // This is true if the version is greater than or equal to the major version it was released in
 // or is greater than or equal to the list of minor releases it was included in.
 func (fr FeatureReleases) IsSupported(versionStr string) bool {
+
+	// in our compatibility tests, our images are in the format of "release-v1.0.x". We want to actually look at
+	// the "1.0.x" part but we also need this to be a valid version. We can change it to "1.0.0"
+	// TODO: change the way we provide the ibc-go version. This should be done in a more flexible way such
+	// as docker labels/metadata instead of the tag, as this will only work for our versioning scheme.
+	const releasePrefix = "release-"
+	if strings.HasPrefix(versionStr, releasePrefix) {
+		versionStr = versionStr[len(releasePrefix):]
+		versionStr = strings.ReplaceAll(versionStr, "x", "0")
+	}
```

Comment:
Would the labels approach be something like adding the `LABEL` cmd to the dockerfile.
Something like:

```
LABEL ibc-go=v7.0.0
LABEL cosmos-sdk=v0.47.0
```

with some smarter way of retrieving the versions. I haven't looked at docker labels a whole lot, there may be a way to add labels after the image is built too

Reply:
yeah that syntax should work, or using `docker build --label`. It's not possible to add labels to an existing container, but they could be re-built as necessary to facilitate this.